scassandra/src/main/scala/com/evolutiongaming/scassandra/Metadata.scala

```diff
@@ -138,6 +138,8 @@ object KeyspaceMetadata {
       val replication = {
         Sync[F].delay { ToScala.from(keyspaceMetadata.getReplication).toMap }
       }
+
+      val userTypes = ToScala.from(keyspaceMetadata.getUserTypes).toList
```

Comment:
`KeyspaceMetadata.getUserTypes` calls may return the different results because the results are build from the mutable collection with outside access, i.e. there is `removeUserType` and `add` methods. It should be hidden behind `Sync` therefore.

Reply:
Agree with you.

P.s Fixed.